src/ui/Forms/Main.cs

```diff
@@ -19089,6 +19109,161 @@ private void ExtendSelectedLinesToPreviousShotChange(bool withGap = false)
             }
         }
 
+        private void SetCueToClosestShotChangeGreenZone(bool isInCue, bool isLeft)
```

Comment:
Wouldn't it be better to have this method somewhere else as the `Main.cs` is already very big with +30k lines which makes the IDE struggle  /cc @niksedk  @Flitskikker

Reply:
@ivandrofly 
Yeah, I noticed the IDE struggling. Although I don't think moving just these ~150 lines will make much difference. I would suggest instead to move all shortcut-related functions to another file.